linuxrc.c

```diff
@@ -895,10 +917,18 @@ void lxrc_init()
   if(!config.had_segv) {
     if (config.linemode)
       putchar('\n');
+#ifdef __s390x__
+    printf(
+      "\n>>> %s installation program v" LXRC_FULL_VERSION " (c) 1996-2019 SUSE LLC on %s <<<\n",
+      config.product,
+      config.hwp.machine_name
+    );
+#else
```

Comment:
That's a lot if `ifdef __s390x__` and copy-and-paste code duplication.

I would suggest something like having a new var like `config.machine` on *all* archs that is initialized with `""` and on s390x gets `" on Foo Bar"`. The string can then be inserted unconditionally in all these `printf`s where it matters.

This also makes it easy later to have something similar on non-s390.